handler.py

```diff
@@ -2,46 +2,42 @@
 from calculations.calculations import Calculations
 from plots.make_plot import make_plot
 import traceback
+import pandas as pd
 
-conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv")
+conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv", "data\\output2_analog_2023-10-14_13-18-15.csv")
 calculations = Calculations("data\\output2_linpot_2023-10-14_13-23-28.csv")
+#conversions.clean_data(conversions.data)
 
 
 class handler:
 
     def wheel_load_handler(self):
         # loads and prepares initial data
-        conversions.convert_voltage_to_in()
-        conversions.convert_time(conversions.data)
-        conversions.clean_data()
-        calculations.data = conversions.data
-
-        # calculating constants
-        calculations.calculate_wheel_rate_front()
-        calculations.calculate_wheel_rate_rear()
-        calculations.calculate_wheel_rate_roll_front()
-        calculations.calculate_wheel_rate_roll_rear()
-        calculations.calculate_K_H_K_P()
-        calculations.calculate_K_R_K_W()
-        calculations.calculate_a_b()
-        calculations.calculate_q_w()
-
+        data_linpot = conversions.data_linpot
+        data_accel = conversions.data_accel
```

Comment:
you can ignore the calls to the accel data since theres anothe portion of client that already takes care of it